docs/sdk.md

```diff
@@ -0,0 +1,96 @@
+Calypso SDK
+===========
+
+Calypso SDK is an early stage tool that's goal is to build, visualize, test, and deliver extensions from a single source.
+
+## Using SDK CLI
+
+You can link Calypso SDK to be globally available by running `npm link` in the Calypso directory. That will give you a global SDK CLI command:
+
+```
+calypso-sdk --help
+```
+
+Alternatively, you can run CLI commands directly in Calypso directory:
+
+```
+npm run sdk -- --help
+```
```

Comment:
at this point I think it would be enough to mention that the alternative calling approach exists and leave only one canonical example. for instance, let's say `npm run sdk` is the preferential calling convention (it's what I've been using but I don't have a strong opinion here) then let's leave only one example way of running these commands to prevent confusion for those who aren't as familiar with the underlying reasons why there are two

```md
Run the SDK CLI commands from within the Calypso directory:
`npm run sdk -- --help`

> Note: It's also possible to run the SDK commands "globally" by linking within the Calypso repository with `npm-link`
> After running this command you can replace all invocations of `npm run sdk` in the examples below with `calypso-sdk` and may do so from any other directory in the filesystem.
```

Reply:
Yep, valid point! 

If we ever rename `calypso-sdk` bin script handle, users would need to run `npm link` to get a symlink with the new handle:
https://github.com/Automattic/wp-calypso/blob/190637cbf23d31db6acdacdd7e45835c098dfd41/package.json#L12-L14

Suppose `npm run` -scripts are going to be quite stable since it's just "sdk" — so they're better suited to be the primary way for now (however ugly that extra `--` is).